README.md

```diff
@@ -0,0 +1,47 @@
+# Pulumi Workshops
+
+This is the official repo for public Pulumi workshops. If a workshop is running from Pulumi, you'll likely find the code here!
+
+Each workshop has a directory. Inside the directory is a README with details about the workshop and what to expect and subdirectories with different labs from the workshop. You generally are not expected to checkout a repo to do a workshop; code provided is for copying and pasting into your own workspace.
```

Comment:
```suggestion
Each workshop has a directory. Inside the directory is a README with details about the workshop and what to expect and subdirectories with different labs from the workshop. You generally are not expected to clone a repo to do a workshop; code provided is for copying and pasting into your own workspace.
```

Reply:
Please feel free to remove the co-authored by if you accept this incredibly picayune edit. Same policy applies for any minor content changes I suggest in the future.